en.lproj/SGLocalizable.strings

```diff
@@ -71,30 +71,30 @@
 "Auth.AccountBackupReminder" = "Make sure you have a backup access method. Keep a SIM for SMS or an additional session logged in to avoid being locked out.";
 "Auth.UnofficialAppCodeTitle" = "You can get the code only with official app";
 
-"Settings.SmallReactions" = "Small reactions";
+"Settings.SmallReactions" = "Small Reactions";
 
 "ContextMenu.SaveToCloud" = "Save to Cloud";
 "ContextMenu.SelectFromUser" = "Select from Author";
 
 "Settings.ContextMenu" = "CONTEXT MENU";
-"Settings.ContextMenu.Notice" = "Disabled entries will be available in \"Swiftgram\" sub-menu.";
+"Settings.ContextMenu.Notice" = "Disabled entries will be available in the Swiftgram sub-menu.";
```

Comment:
It now sounds more ambiguous. Somebody may think they need to go to Settings > Swiftgram to find these buttons. Please, revert.